packages/core/src/components/button/abstractButton.tsx

```diff
@@ -29,6 +29,12 @@ export interface IButtonProps extends IActionProps {
     loading?: boolean;
 
     /**
+     * If set to `true`, the button will display in an active state. This is equivalent to setting `pt-active` via className.
```

Comment:
@DrewDennison the Blueprint pattern is something like your second offering above:

![image](https://cloud.githubusercontent.com/assets/443450/23191523/f0e3ee36-f852-11e6-91c5-0e983d0c7fc9.png)

 "Whether the button is active." is probably the most in line with the above pattern, but then it's not necessarily clear what "active" means at a glance. @giladgray may have thoughts.